AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginLazyLoadTests/LL12/HasOneParentChild/AWSDataStoreLazyLoadHasOneTests.swift

```diff
@@ -22,13 +22,13 @@ class AWSDataStoreLazyLoadHasOneTests: AWSDataStoreLazyLoadBaseTest {
     func testSaveHasOneParent() async throws {
```

Comment:
i think we need to add given when then statements for integration tests as decided before.

Reply:
oh okay, could you please add for the newly added ones?